CHANGELOG.md

```diff
@@ -4,9 +4,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## 0.1.3 - 20178-02-08
+## 0.1.3 - 2018-02-08
 Initial public release.
 
 ### Added
 * The `ngx-uploadcare-widget` component.
-* README with info about install, usage, etc.
\ No newline at end of file
+* README with info about install, usage, etc.
+
+## 0.1.4 - 2018-03-12
+Interaction release
+
+### Added
+* Now component is interactive, widget accept changes of input parameters.
+* Added `clearUploads()` method to remove current ulpoads.
+* Added `reset()` method reset widget mannualy.
+* Updated README with info about install, usage, etc.
+* Updated Demo projects with examples of usage
```

Comment:
New releases must be added from above.
Read more about the format of the changelog here - https://keepachangelog.com/en/1.0.0/

Reply:
You need something like

```
### Added

* New `clearUploads()` method ...
* New `reset()` method

### Changed

* Now component is interactive, widget accept changes of input parameters.
....
```